modules/api/php/endpoints/candidates.class.inc

```diff
@@ -154,6 +165,12 @@ class Candidates extends Endpoint implements \LORIS\Middleware\ETagCalculator
         $provisioner = (new \LORIS\api\Provisioners\CandidatesProvisioner())
             ->filter($filter);
 
+        if ($this->useProjectFilter()) {
```

Comment:
Why add a function that always return true?

Reply:
I took this part from the example you suggested https://github.com/aces/Loris/blob/main/modules/candidate_list/php/candidate_list.class.inc#L59-L62. I thought maybe it was to be completed and it would become meaningful, but I don't see a case where filters would not be used, so I'm removing it.